src/index.html

```diff
@@ -5,8 +5,443 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo">
+        <img class="logo"
+         src="/images/logo.png"
+         alt="logo"><a href="#index.html"></a>
+        </img>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#laptops&computers"
+            class="nav__link" data-qa="nav-hover">laptops & computers</a>
```

Comment:
pay attention to your indentation
```suggestion
            <a 
              href="#laptops&computers"
              class="nav__link" 
              data-qa="nav-hover"
            >
              laptops & computers
            </a>
```